### PR TITLE
Update logging to handle CMA parameter configured events

### PR DESCRIPTION
**Pull request by Jkovarik**
This PR addresses changes from https://github.com/nasa/cumulus/pulls in Cumulus Core as well as nasa/cumulus-message-adapter#58

### CODE REVIEW

File: message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java [hunk 1]
```diff
@@ -205,6 +205,31 @@ public void testException()
...
+    public void testLoggerWithCmaConfiguration() throws MessageAdapterException
+    {
+        final LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
+        final Configuration config = ctx.getConfiguration();
+        TestAppender appender = (TestAppender) config.getAppenders().get("TestAppender");
+
+        MessageParser parser = new MessageParser(new TestEventMessageAdapter());
+        try
+        {
+            String inputJsonString = AdapterUtilities.loadResourceToString("parameter.input.json");
```
Comment (nemreid) [inline message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java:219]: I'm not seeing the `parameter.input.json` in this PR or elsewhere in the code base?

**Review by nemreid — CHANGES_REQUESTED** [review]
Seems to be missing a test input file.

File: message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java (hunk 1 above)
Comment (Jkovarik) [reply message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java:219]: That would help to have it included, yes 👍

Comment (Jkovarik) [reply message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java:219]: 550cde1

**Review by nemreid — APPROVED** [review]
👍

**Pull request merged**